cspell.json

```diff
@@ -151,16 +176,21 @@
         "shizzle",
         "Shopify",
         "Smorgeni",
+        "sourcecode",
         "Spinbuttons",
         "Starkrimson",
         "Studienska",
+        "stylelint",
+        "subfolders",
         "submenu",
         "submenu's",
         "submenus",
         "Sulaiman",
         "tabbable",
-        "tabindex",
+        "tablist",
+        "tablist’s",
```

Comment:
Actually, I think I'd rather not add `tablist’s` and instead just change the fancy Unicode U+2019 apostrophe to a regular old ASCII 0027 apostrophe in the 2 tabs.js files?

i.e. change `tablist’s` to `tablist's` in the tabs.js code comments

I assume cSpell will then just ignore `tablist's` without needing it to be added to the words list?

Reply:
I think the contraction needs to be excluded as well, but I can get rid of the smart quote

Reply:
Let me know - I'm curious. I see lots of instances of e.g. `button's`, `editor's`, `page's`, `device's`, `element's`, `user's` and other possessive nouns in the APG, and they're not listed in the words list. I wonder if some combination of the compound words algorithm followed by `'s` causes some possessives to need special treatment. Now that we're not allowing compound words to slip through the cracks, I'm guessing you can remove all of the possessives from the words list?

Reply:
Just tried a find/replace with the apostrophe to single quote, but it still needs the word in the dictionary. I think that's because the original/root word isn't known. I won't add that conversion to this PR to keep the diff noise down, but may be worth doing after this lands as a separate thing

Reply:
Right - might have to add "tablist" before experimenting with removing "tablist's".

If you do try it at some point, I'm also curious about this triple in the words list:
        "submenu",
        "submenu's",
        "submenus",

Wondering if only "submenu" is needed after allowCompoundWords is turned off.